modules/Sources/Features/Root/RootInitialization.swift

```diff
@@ -165,10 +182,8 @@ extension RootReducer {
                 /// We need to fetch data from keychain, in order to be 100% sure the keychain can be read we delay the check a bit
                 return .concatenate(
                     Effect.send(.initialization(.configureCrashReporter)),
-                    .run { send in
-                        try await mainQueue.sleep(for: .seconds(0.02))
-                        await send(.initialization(.checkWalletInitialization))
-                    }
+                    // TODO: anything that checks or required seed is wrong here
```

Comment:
Should this be "that checks for required seed", or "that checks or requires the seed"?

Reply:
Removed completely, I think it was just for me during debugging session. You are right, it was supposed to be `requires` instead of required. The rule says any TODO must be followed by an issue number so the right format is TODO: [#.... That leads me to conclusion it is a debug todo just for me and shouldn't be in the final PR.